customer-commons/src/main/java/no/unit/nva/customer/validator/ChannelClaimValidator.java

```diff
@@ -22,12 +31,39 @@ public static void validate(ChannelClaimDto channelClaim) throws BadRequestExcep
         if (isNull(channelClaim) || isNull(channelClaim.channel())) {
```

Comment:
If you want you can wrap in single optional instead of two null checks:
`Optional.ofNullable(channelClaim).map(ChannelClaimDto::channel).isEmpty()`